src/components/ItemCard.js

```diff
@@ -0,0 +1,199 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import CardSlide from './CardSlide';
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+
+const ItemCard = () => {
+  const [roomData, setRoomData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/RoomData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => setRoomData(data));
+  });
+
+  return (
+    <Card>
+      {roomData.map(data => {
```

Comment:
```suggestion
      {roomData.map(({ name, address, description, bed, host_id, price }) => {
```
여기는 이렇게 구조분해할당해서 사용할 수 있겠네요!
그렇게 되면 가독성은 조금 더 높아질 수 있습니다!